### PR TITLE
add test files for parse2naf and nafdocument

### PR DESCRIPTION
**Pull request by hmduerinck**

### CODE REVIEW

**Review by LucSevereijns — APPROVED** [review]

File: tests/test_nafdocument.py [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+unittest.TestLoader.sortTestMethodsUsing = None
+
+
+class TestNafDocument(unittest.TestCase):
+    """
+    The basic class that inherits unittest.TestCase
+    """
+
+    def test_generate():
+        '''
```
Comment (royklip) [inline tests/test_nafdocument.py:15]: Why not use """ here like the class description?

File: tests/test_nafdocument.py [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+        This function tests whether the naf document initalization is done correctly
+        input: etree._ElementTree + dict
+        level: 2
+        scenarios: check added features vs input
+        '''
```
Comment (royklip) [inline tests/test_nafdocument.py:20]: You used "pass" in the other PR, you could add them in these tests as well

File: tests/test_nafdocument.py [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+unittest.TestLoader.sortTestMethodsUsing = None
+
+
+class TestNafDocument(unittest.TestCase):
+    """
+    The basic class that inherits unittest.TestCase
+    """
+
+    def test_generate():
```
Comment (royklip) [inline tests/test_nafdocument.py:14]: You could add the "self" argument in advance

**Review by royklip — COMMENTED** [review]
All

File: tests/test_nafdocument.py (hunk 3 above)
Comment (hmduerinck) [reply tests/test_nafdocument.py]: done

**Review by royklip — APPROVED** [review]

**Pull request merged**